modules/util/string.go

```diff
@@ -4,7 +4,9 @@
 
 package util
 
-import "github.com/yuin/goldmark/util"
+import (
+	"github.com/yuin/goldmark/util"
+)
```

Comment:
?

Reply:
formatting change. Irrelevant and not worth worrying about.